draft-link-v6ops-464xlaton.xml

```diff
@@ -121,9 +121,18 @@ This document complements <xref target="RFC6877"/> by  providing recommendations
 <name>
 Terminology
 </name>
+<ul>
+<li>
 <t>
-To be added
+IPv6-only network: a network which do not assign IPv4 addresses to hosts and provides connectivity to IPv4-only destinations via NAT64 (<xref target="RFC6146"/>).
```

Comment:
```suggestion
IPv6-only network: a network which does not assign IPv4 addresses to hosts and may provide connectivity to IPv4-only destinations via NAT64 (<xref target="RFC6146"/>).
```

Per Lorenzo's comment: just because a network is IPv6-only does not mean it also provides NAT64 necessarily.

Reply:
That's actually interesting. I read his comment as we need to make it explicit that this draft is ONLY applicable for NAT64 networks. That's why I said 'provides'.
Maybe we shall add a text saying 'As 464XLAT architecture requires the network-side translation in a form of NAT64, this draft is only applicable to IPv6-only networks providing such translations.'
How about that?

Reply:
Ah I see your point. Yes, that would be good. Alternatively, we could define a more specific term for this draft, such as "NAT64-enabled network" that cuts to the chase? Either is fine with me, I just was convinced that we should not try to define "IPv6-only network" as exclusive to NAT64-supporting networks.

Reply:
Let me suggest the text. two mins!

Reply:
LGTM